<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorColorWidget</class>
 <widget class="QWidget" name="InspectorColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutColor">
     <item>
      <spacer name="horizontalSpacerColor">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelColor">
       <property name="text">
        <string>&lt;a href=&quot;Color&quot; style=&quot;color: #BEBEBE;&quot;&gt;Color&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditColor"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonColor">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/Graphics/Images/Color.png</normaloff>:/Graphics/Images/Color.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransition">
     <item>
      <spacer name="horizontalSpacerTransition">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransition">
       <property name="text">
        <string>&lt;a href=&quot;Transition&quot; style=&quot;color: #BEBEBE;&quot;&gt;Transition&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTransition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDuration">
     <item>
      <spacer name="horizontalSpacerDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDuration">
       <property name="text">
        <string>&lt;a href=&quot;Duration&quot; style=&quot;color: #BEBEBE;&quot;&gt;Duration&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTween">
       <property name="text">
        <string>&lt;a href=&quot;Tween&quot; style=&quot;color: #BEBEBE;&quot;&gt;Tween&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDirection">
     <item>
      <spacer name="horizontalSpacerDirection">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>34</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDirection">
       <property name="text">
        <string>&lt;a href=&quot;Direction&quot; style=&quot;color: #BEBEBE;&quot;&gt;Direction&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDirection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutUseAuto">
     <item>
      <spacer name="horizontalSpacerUseAuto1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelUseAuto">
       <property name="text">
        <string>&lt;a href=&quot;Use auto&quot; style=&quot;color: #BEBEBE;&quot;&gt;Use auto&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxUseAuto">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerUseAuto2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerColor">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBoxDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>durationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTransition</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>transitionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxDirection</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>directionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTransition</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>resetTransition(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDuration</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>resetDuration(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTween</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>resetTween(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDirection</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>resetDirection(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelUseAuto</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>resetUseAuto(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseAuto</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorColorWidget</receiver>
   <slot>useAutoChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>durationChanged(int)</slot>
  <slot>loopChanged(int)</slot>
  <slot>transitionChanged(QString)</slot>
  <slot>directionChanged(QString)</slot>
  <slot>seekChanged(int)</slot>
  <slot>lengthChanged(int)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>resetTransition(QString)</slot>
  <slot>resetDuration(QString)</slot>
  <slot>resetTween(QString)</slot>
  <slot>resetDirection(QString)</slot>
  <slot>resetSeek(QString)</slot>
  <slot>resetLength(QString)</slot>
  <slot>resetLoop(QString)</slot>
  <slot>pauseOnLoadChanged(int)</slot>
  <slot>resetPauseOnLoad(QString)</slot>
  <slot>resetUseAuto(QString)</slot>
  <slot>useAutoChanged(int)</slot>
 </slots>
</ui>
